etc/kayobe/ansible/octavia-amphora-image-register.yml

```diff
@@ -39,7 +40,14 @@
         checksum_algorithm: md5
       changed_when: false
       register: image_checksum
-      when: image_info.openstack_image
+      when: image_info.image
+
+    - name: Assert that Amphora image exists
+      assert:
+        that: image_checksum.stat.exists
```

Comment:
The image_checksum variable is registered by a conditional task.

Reply:
Good spot - thanks.